Samples/ApiExamples/ApiExamples.Mac/ApiExamples.Mac.csproj

```diff
@@ -27,6 +25,7 @@
     <CodeSigningKey>Mac Developer</CodeSigningKey>
     <EnableCodeSigning>false</EnableCodeSigning>
     <CreatePackage>false</CreatePackage>
+    <UseRefCounting>false</UseRefCounting>
```

Comment:
Sorry - just to be clear the ":/" was me feeling sad if these changes are really needed. I was kind of hoping the new "UseRefCounting" GC would work now after the changes we made to the binding context recently.